Configurations/10-main.conf

```diff
@@ -810,6 +810,16 @@ my %targets = (
         perlasm_scheme   => "64",
         multilib         => "64",
     },
+     "linux64-loongarch64" => {
+         inherit_from     => [ "linux-generic64" ],
+         cflags           => add("-mabi=lp64"),
+         cxxflags         => add("-mabi=lp64"),                                                           
```

Comment:
This is old-world toolchain convention, right? According to [the current toolchain conventions spec](https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html), there is no `-mabi=lp64`, only its fully-qualified variants, you may mean `-mabi=lp64d` here as it's the most common ABI.

Reply:
Also you should not put LoongArch below MIPS, as these are entirely different architectures (the Loongson Corporation repeatedly says they are entirely different, right?); most likely you should place the block alphabetically and add a section header like the riscv64 one.

Reply:
Sorry to reply you so late. Yes，this is my amendment last year。I will fix it